static_src/stores/user_store.js

```diff
@@ -61,6 +61,20 @@ export class UserStore extends BaseStore {
         break;
       }
 
+      case userActionTypes.USER_INVITE_STATUS_UPDATED: {
```

Comment:
Do we eventually want to have the similar functionality for inviting users to spaces? If so, we should pass an additional parameter to be able to determine which level (org vs space) are we inviting a user. I'm also fine with leaving it simple like this but just change the action to something like USER_ORG_INVITE_STATUS_UPDATED so that we don't handle the conditional logical in this case statement and we will later create a USER_SPACE_INVITE_STATUS_UPDATED action and just handle which one to send in `receivedInviteStatus` by passing an additional argument there.

after i just typed that, i'm leaning towards the latter.

with all that said, this PR is just for the org but would want it in a way so we can easily throw the space one together if we do plan on that.

thoughts?